src/pages/utilities/UnrecognizedUtilityTabs/UnrecognizedTab.tsx

```diff
@@ -117,6 +117,7 @@ const Menu = (
         const failedCount = countBy(result, 'status').rejected;
         if (failedCount) toast.error(`Error ignoring ${failedCount} files!`);
         if (failedCount !== selectedRows.length) toast.success(`${selectedRows.length} files ignored!`);
+        invalidateQueries(['files']);
```

Comment:
This is not needed now since the invalidation on mutation itself has changed. This is duplicate invalidation

Reply:
Did you try? Without this, unrecognized does not update, you have to refresh.

Reply:
yes

https://github.com/ShokoAnime/Shoko-WebUI/assets/26010946/ef10066d-91af-48b6-a465-508b92f50cbd